ttnn/cpp/ttnn/operations/conv2d/conv2d.cpp

```diff
@@ -296,12 +304,16 @@ std::tuple<ttnn::Shape, ttnn::MemoryConfig, bool> get_conv_padded_input_shape_an
                 if (conv_config.core_grid.value() != input_shard_grid) {
                     needs_shard_or_reshard = true;
                 }
-                bool input_tensor_height_sharded = input_shard_scheme == TensorMemoryLayout::HEIGHT_SHARDED;
-                if (conv_config.height_sharding != input_tensor_height_sharded) {
+                if(conv_config.shard_layout!=input_shard_scheme)
+                {
                     needs_shard_or_reshard = true;
                 }
+                // bool input_tensor_height_sharded = input_shard_scheme == TensorMemoryLayout::HEIGHT_SHARDED;
```

Comment:
should we remove this block of commented out code?

Reply:
Removed